packages/cli/config/eslint.backend.js

```diff
@@ -46,6 +46,8 @@ module.exports = {
         bundledDependencies: true,
       },
     ],
+    'no-unused-expressions': 'off',
```

Comment:
wat

Reply:
This is so you can do `Things?.like?.this` in typescript without the linter throwing a BF.

we need to use `'@typescript-eslint/no-unused-expressions': 'error',` instead.

Reply:
Oh you enabled the TS one, didnt catch that, :+1: 

Something for web-scripts @fastfrwrd?